math/cmath/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ algebra_add_library(algebra_cmath_math cmath_math
    "include/algebra/math/impl/cmath_operators.hpp"
    "include/algebra/math/impl/cmath_transform3.hpp"
    "include/algebra/math/impl/cmath_vector.hpp"
+   "include/algebra/math/impl/cmath_vector_soa.hpp"
```

Comment:
I don't see this file in this PR. :confused: (:thinking: we may want to make it an error when we point at non-existent files with CMake...)